platform-server-rest/src/main/java/uno/cod/platform/server/rest/controller/ChallengeController.java

```diff
@@ -21,17 +22,20 @@ public ChallengeController(ChallengeService service) {
     }
 
     @RequestMapping(value = RestUrls.CHALLENGES, method = RequestMethod.POST)
+    @PreAuthorize("isAuthenticated() and @securityService.isOrganizationAdmin(principal, #dto.organizationId)")
```

Comment:
any user can be organization member but he cannot see all the challegnes for one organization. Just the organization admin can see them.